lib/parallel_tests/cli.rb

```diff
@@ -190,12 +190,20 @@ def parse_options!(argv)
           "Do not run any other tests in the group used by --single(-s)") do |pattern|
 
           options[:isolate] = true
+          options[:isolate_count] = 1
```

Comment:
prefer not setting this here since it makes the ruby api less usable if this is assumed to always be set